presto-main/src/main/java/com/facebook/presto/sql/planner/plan/ExchangeNode.java

```diff
@@ -13,6 +13,7 @@
  */
 package com.facebook.presto.sql.planner.plan;
 
+import com.facebook.presto.sql.planner.OrderingScheme;
```

Comment:
please add proper `OrderingSchema` formatting support to `PlanPrinter.Visitor#visitExchange`